### PR TITLE
LPS-158153 Add new functionality to SystemProperties: get property with default value, and get array

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , This PR added two new methods to the SystemProperties.java

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me scan test results when you free ? Thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

File: portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import org.junit.Test;
+
+/**
+ * @author Jiaxu Wei
+ */
+public class SystemPropertiesTest {
+
+	@Before
+	public void setUp() {
+		Map<String, String> properties = ReflectionTestUtil.getFieldValue(
```
Comment (dantewang) [inline portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java:33]: @W2jx Let's update the key & update logic.

Create a prefix for the keys, and in each test case, every time you need a unique key, use the prefix and a random string created using `RandomTestUtil`.

Then, change `setUp` into `@After tearDown` and remove properties with the prefix in the `tearDown`.

**Comment by dantewang** [comment]
@W2jx Please see my comment and resend. Thanks!

File: portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java (hunk 1 above)
Comment (W2jx) [reply portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java:33]: I'll fix it.

**Pull request closed without merging**